pkg/cmd/pr/checks/checks.go

```diff
@@ -66,6 +66,11 @@ func NewCmdChecks(f *cmdutil.Factory, runF func(*ChecksOptions) error) *cobra.Co
 
 			Without an argument, the pull request that belongs to the current branch
 			is selected.
+
+			Exit codes:
+		      0: All checks passed
+		      1: One or more checks failed
+		      8: Checks pending
```

Comment:
We shouldn't be changing the exit code meanings here, just adding the exit code 8 information:

```suggestion
			Additional exit codes:
		      8: Checks pending
```